src/components/explorer/GuildCard.tsx

```diff
@@ -35,7 +35,9 @@ const GuildCard = ({ guildData }: Props): JSX.Element => (
         gap={4}
         alignItems="center"
       >
-        {image && <GuildLogo imageUrl={guildData.imageUrl} size={"48px"} />}
+        {image && (
+          <GuildLogo imageUrl={guildData.imageUrl} size={"48px"} imageQuality={2} />
```

Comment:
Is the `imageQuality={2}` attribute intentional here? That number seems way too low for me. 
Also, the `size` prop in `GuildLogo` is `"48px"` by default, so I think we don't need to define it here.